app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -0,0 +1,33 @@
+package com.example.parkingsystemjava.mvp.view;
+
+import android.app.Activity;
+import android.app.FragmentManager;
+import android.content.Context;
+import android.widget.Toast;
+import com.example.parkingsystemjava.R;
+import com.example.parkingsystemjava.fragments.SpacesSettingDialogFragment;
+import com.example.parkingsystemjava.listener.ListenerDialogFragment;
+import com.example.parkingsystemjava.mvp.contract.ParkingContract;
+import com.example.parkingsystemjava.mvp.view.base.ActivityView;
+
+public class ParkingView extends ActivityView implements ParkingContract.MainActivityView {
+
+    public ParkingView(Activity activity) {
+        super(activity);
+    }
+
+    @Override
+    public void showDialogFragment(String parkingLots, ListenerDialogFragment listenerDialogFragment) {
```

Comment:
this parking lots are not used please delete

Reply:
done